x-pack/test/cloud_security_posture_functional/page_objects/csp_dashboard_page.ts

```diff
@@ -82,6 +83,35 @@ export function CspDashboardPageProvider({ getService, getPageObjects }: FtrProv
       }
     },
 
+    getAllComplianceScoresByCisSection: async (tab: DashboardTabs) => {
+      await dashboard.getDashoard(tab);
+      const pageContainer = await testSubjects.find('pageContainer');
+      return await pageContainer.findAllByTestSubject('cloudSecurityFindingsComplianceScore');
+    },
+
+    getDashoard: async (tab: DashboardTabs) => {
+      if (tab === 'Cloud') {
```

Comment:
Capitalised keys are quite unusual for JS/TS, I'd have them as constants or as enum
```
const CLOUD = 'cloud';
const KUBERNETES = 'kubernetes';

or smth like

const TAB_TYPES = {
  CLOUD: 'cloud',
  KUBERNETES: 'kubernetes'
} as const;
```